### ISSUE TITLE
Toggle Oled on TTGO boards

### ISSUE THREAD

**Issue by pauldogg** [sudomesh/disaster-radio#102]
Ability to toggle the oled to save battery life.
I'll be putting mine in an weatherproof enclosure; ideally i'd like it if there's a on/off pin I can use to turn the oled off if the box is open/closed.

Thanks

**Comment by paidforby** [comment]
Good idea. Should be possible to add this feature to the existing console settings system. changes would need to be made here,

https://github.com/sudomesh/disaster-radio/blob/master/firmware/src/middleware/Console.cpp#L246
https://github.com/sudomesh/disaster-radio/blob/master/firmware/src/settings/settings.cpp

I'm guessing it would require a reboot to take effect.
